packages/fluentui/react-bindings/src/context-selector/useContextSelectors.ts

```diff
@@ -54,7 +54,7 @@ export const useContextSelectors = <
 
       try {
         const statePayloadHasChanged = Object.keys(prevState[0]).some((key: Properties) => {
-          return !Object.is(prevState[0][key] as SelectedValue, payload[1][key]);
+          return !Object.is(prevState[0][key], (payload[1] as Record<string, any>)[key]);
```

Comment:
```suggestion
          return !Object.is(prevState[0] /* previous contextValue */[key], payload[1] /* current contextValue */[key]);
```

If we will modify typings on line 11, we can remove `as` there:

```diff
-  Value,
+  Value extends Record<string, any> /* should be an object always */,
```